web-app/js/portal/cart/DownloadPanelBody.js

```diff
@@ -119,9 +119,19 @@ Portal.cart.DownloadPanelBody = Ext.extend(Ext.Panel, {
         params.onAccept = function(callbackParams) {
             var downloader = new Portal.cart.Downloader({
                 listeners: {
-                    'downloadrequested': function(downloadUrl) { log.debug('Download requested', downloadUrl); },
-                    'downloadstarted': function(downloadUrl) { log.debug('Download started', downloadUrl); },
-                    'downloadfailed': function(downloadUrl, msg) { log.debug('Download failed', downloadUrl, msg); }
+                    'downloadrequested': function(downloadUrl) {
+                        log.debug('Download requested', downloadUrl);
+                    },
+                    'downloadstarted': function(downloadUrl) {
+                        log.debug('Download started', downloadUrl);
+                    },
+                    'downloadfailed': function(downloadUrl, msg) {
+                        Ext.Msg.alert(
+                            OpenLayers.i18n('errorDialogTitle'),
+                            OpenLayers.i18n('downloadErrorText')
+                        );
+                        log.debug('Download failed', downloadUrl, msg);
```

Comment:
Go with error here. I think down the track we might want to actually parse this.